{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/{{cookiecutter.app_name}}/management/assign_superuser_status.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management import BaseCommand
```

Comment:
I couldn't get this command to work initially - what fixed it was adding a 'commands' directory, so the structure was [package_name]/[app_name]/management/commands/assign_superuser_status.py

Maybe it would also help to add text like this to the README: https://github.com/i-dot-ai/one-big-thing/#how-to-access-the-admin (although the command flags are slightly different)?